templates/expo-template-tabs/navigation/useWebLinking.js

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import { Platform } from 'react-native';
+import { useLinking } from '@react-navigation/native';
+
+export default function useWebLinking(containerRef) {
+  if (Platform.OS === 'web') {
```

Comment:
I think we should have this for both web and native

Reply:
Also would be good to add a link to the deep linking docs above

Reply:
I don't think we need to have deep linking in an example app. Do you find it needed?

Reply:
Why not? Also otherwise it's confusing why this code is only for web.